egg/core/continous_communication.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import Callable
+
+import torch.nn as nn
+import torch.nn.functional as F
+
+from egg.core import Interaction
+
+
+class ContinuousLinearSender(nn.Module):
+    def __init__(
+        self,
+        agent: nn.Module,
+        encoder_input_size: int,
+        encoder_hidden_size: int = 64,
+        num_layers: int = 1,
```

Comment:
i think having a multi-layer option is a bit of an overkill, probably this should only be used for debugging/simple baselining. What do you think?

Reply:
I'd be in favour of keeping it (with one layer as default) just in case. Also because I guess RNN with multistep unfolding and continuous output has a straight comparison with a multilinear communication channel